sys/include/phydat.h

```diff
@@ -79,6 +79,8 @@ enum {
     UNIT_TEMP_C,    /**< degree Celsius */
     UNIT_TEMP_F,    /**< degree Fahrenheit */
     UNIT_TEMP_K,    /**< Kelvin */
+    /* luminosity */
```

Comment:
shouldn't it rather be called something like `Illuminance`? `luminosity` is IMO related to `Luminance (candela/m²)`, which does not equal `Illuminance (lux)`